Files/Views/MainPage.xaml.cs

```diff
@@ -492,5 +501,54 @@ private void HorizontalMultitaskingControl_Loaded(object sender, RoutedEventArgs
         {
             MultitaskingControl = HorizontalMultitaskingControl;
         }
+
+        private static string GetDriveTypeIcon(DriveInfo drive)
```

Comment:
Does this has to be in MainPage.cs?

Reply:
It can be moved to another page since it is a static function.  Please let me know if the function needs to be moved to another page.

Reply:
That would be best.

Reply:
Okay. I'll move it somewhere else. Is there a specific file where you would have it placed? Or is the Interaction.cs file okay?

Reply:
@tsvietOK Any ideas?

Reply:
I think we could keep it as it is, as it is used here.  By the way, SetSelectedTabInfoAsync should be moved too.

Reply:
It's best if it's moved so that we can reuse it, I meant ideas where to move it to.